packages/ember-htmlbars/lib/helpers/each.js

```diff
@@ -76,7 +76,7 @@ export default function eachHelper(params, hash, blocks) {
   var keyPath = hash.key;
 
   if (blocks.template.arity === 0) {
-    Ember.deprecate(deprecation);
+    Ember.deprecate(deprecation, false, { id: 'ember-htmlbars.each-helper-arity', until: '3.0.0' });
```

Comment:
Let's mark this as 2.0.0 and create an issue for cleaning it up. 

The only scenario where this should be hit is when using context switching each, and we need to remove support (I thought we already did actually) in 2.0.0.